docs/source/understanding.rst

```diff
@@ -10,4 +10,5 @@ also of considerations for deploying Calico at large scale.
    l2-interconnectFabric
    arch-felix-and-acl
    calico-neutron-api
-   api-proposal
+   etcd-architecture
```

Comment:
As above, at this point we might want to put the etcd stuff in a "Futures" section, then move it here when we actually merge etcd into master.